Sample Test-network build fails on ecc docker build

**Describe the bug**
<!-- 
  Add a clear and concise description of what the bug is. 
-->
In a vanilla installation when you want to run the test-network sample the readme says to run make build from the test-network directory. That make build never completes. It gets stuck when it builds the ecc docker container.
**To Reproduce**
<!-- 
Add steps to reproduce the behavior, e.g.
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
-->
1. Go to utils/docker and execute make run
2. from FPC_ROOT_DIRECTORY run make build
3. go to samples/deployment/test-network and execute first step in readme which exports 2 environment variables and executes make build
4. Build process starts and gets stuck at "docker  build --no-cache --build-arg FPC_VERSION=main  --build-arg CAAS_PORT=9999 --build-arg CC_NAME="ecc" --build-arg CC_PATH="/usr/local/bin" --build-arg CC_LIB_PATH="/usr/local/bin/enclave/lib" -t hyperledger/fabric-private-chaincode-boilerplate-ecc -f Dockerfile.boilerplate-ecc"
**Expected behavior**
<!-- 
  Add a clear and concise description of what you expected to happen. 
-->
make build should complete successfully
**Log-filesScreenshots**
<!-- 
 If applicable, add screenshots to help explain your problem. 
-->

**Environment (please complete the following information):**
<!--
 - OS (distro+version): [e.g. Ubuntu 20.04]
 - FPC Version (tag or commit-id) [e.g. 'concept-release']
-->
OS: Ubuntu 20.04
FPC Version: main branch
**Additional context**
<!--
  Add any other context about the problem here.
-->

@mbrandenburger after updating docker, restarting, executing docker system prune --force, and going starting the fpc environment I get the following error:
`root@docker-desktop:/project/src/github.com/hyperledger/fabric-private-chaincode# cd $FPC_PATH/samples/deployment/test-network
root@docker-desktop:/project/src/github.com/hyperledger/fabric-private-chaincode/samples/deployment/test-network# export TEST_CC_ID=echo
root@docker-desktop:/project/src/github.com/hyperledger/fabric-private-chaincode/samples/deployment/test-network# export TEST_CC_PATH=${FPC_PATH}/samples/chaincode/echo
root@docker-desktop:/project/src/github.com/hyperledger/fabric-private-chaincode/samples/deployment/test-network# make -C $FPC_PATH/utils/docker build
make: Entering directory '/project/src/github.com/hyperledger/fabric-private-chaincode/utils/docker'
DOCKER_BUILDKIT=1 docker  build  --build-arg FPC_VERSION=main  -t hyperledger/fabric-private-chaincode-base-rt base-rt
failed to dial gRPC: cannot connect to the Docker daemon. Is 'docker daemon' running on this host?: dial unix /Users/mmunaro/.docker/run/docker.sock: connect: no such file or directory
make: *** [Makefile:175: base-rt] Error 1
make: Leaving directory '/project/src/github.com/hyperledger/fabric-private-chaincode/utils/docker'
root@docker-desktop:/project/src/github.com/hyperledger/fabric-private-chaincode/samples/deployment/test-network# `

--- a/samples/deployment/test-network/README.md
+++ b/samples/deployment/test-network/README.md
@@ -15,6 +15,7 @@ Use `CC_ID` and `CC_PATH` to define the FPC Chaincode you want to build.
 cd $FPC_PATH/samples/deployment/test-network
 export CC_ID=echo
 export CC_PATH=${FPC_PATH}/samples/chaincode/echo
+make -C $FPC_PATH/utils/docker build
 make build
 ```
 

--- a/samples/deployment/test-network/setup.sh
+++ b/samples/deployment/test-network/setup.sh
@@ -65,7 +65,7 @@ else
 fi
 echo "set FPC_PATH_HOST = ${FPC_PATH_HOST}"
 
-FABRIC_SAMPLES_HOST=${FPC_PATH}/samples/deployment/test-network/fabric-samples
+FABRIC_SAMPLES_HOST=${FPC_PATH_HOST}/samples/deployment/test-network/fabric-samples
 
 echo "Resolving relative docker volume paths in ..."
 
@@ -90,7 +90,7 @@ done
 
 echo "Preparing blockchain explorer"
 BE_PATH="${FPC_PATH}/samples/deployment/test-network/blockchain-explorer"
-BE_PATH_HOST="${FPC_PATH_HOST}/samples/deployment/test-network/${BE_PATH}"
+BE_PATH_HOST="${FPC_PATH_HOST}/samples/deployment/test-network/blockchain-explorer"
 BE_CONFIG=${BE_PATH}/config.json
 BE_CONNECTIONS_PROFILE=${BE_PATH}/connection-profile/test-network.json
 BE_DOCKER_COMPOSE="${BE_PATH}/docker-compose.yaml"

